### PR TITLE
Fix coverage runs

### PR DESCRIPTION
**Pull request by dothebart**
- it will use on python run to run cluster and single server
- coredump compressing is now utilizing all cpus available
- it wil turn state to orange (jenkins jobs need to be configured to react to exit 5 hovever)
- I tried to get the gcov error messages lower, however, it seems that the current implementation doesn't properly work at all, but due to later issues.
- locating of source files for generating jenkins UI reports have been fixed
- enterprise tests filtering has been fixed

### CODE REVIEW

**Review by naushniki — APPROVED** [review]
@dothebart Good job!

**Review by KVS85 — CHANGES_REQUESTED** [review]

File: jenkins/runCoverageRocksDB.fish [hunk 1]
```diff
@@ -18,13 +18,16 @@ and begin
   if test -d $WORKDIR/work/gcov ; mv $WORKDIR/work/gcov $WORKDIR/work/gcov.old ; end
 
   rocksdb
-  single     ; oskarFull --isAsan true --sanitizer true ; or set s $status
-  cluster    ; oskarFull --isAsan true --sanitizer true ; or set s $status
-
+  both     ; oskarFull --isAsan true --sanitizer true ; or set s $status
+  echo "S: $s"
+  echo $status
```
Comment (goedderz) [inline jenkins/runCoverageRocksDB.fish:23]: Debugging leftover?
```suggestion
```

File: jenkins/runCoverageRocksDB.fish [hunk 2]
```diff
@@ -18,13 +18,16 @@ and begin
...
   collectCoverage
-  or set s $status
 end
-or set s $status
+or set exitcode $status
 
-cd "$HOME/$NODE_NAME/$OSKAR" ; moveResultsToWorkspace ; unlockDirectory 
-exit $s
+cd "$HOME/$NODE_NAME/$OSKAR" ; moveResultsToWorkspace ; unlockDirectory
+echo "exiting $exitcode"
```
[the remark this answers is not in the document]
Comment (dothebart) [reply jenkins/runCoverageRocksDB.fish:30]: I would like to leave this one, so one has a clear indicator.

File: jenkins/runCoverageRocksDB.fish [hunk 3]
```diff
@@ -19,10 +19,14 @@ and begin
...
   cluster    ; oskarFull --isAsan true --sanitizer true ; or set s $status
-
-  collectCoverage
-  or set s $status
+  echo "S: $s"
+  echo $status
+  if test $status -eq 0
+     collectCoverage
+     or set s $status
+  end
```
[the remark this answers is not in the document]
Comment (dothebart) [reply jenkins/runCoverageRocksDB.fish]: changed accordingly.

**Comment by dothebart** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/27300/ devel run
https://jenkins01.arangodb.biz/job/arangodb-devel-coverage/494/ coverage

**Comment by dothebart** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/27320/
https://jenkins01.arangodb.biz/job/arangodb-devel-coverage/496/

**Comment by dothebart** [comment]
https://jenkins01.arangodb.biz/job/arangodb-3.10-coverage/53/
https://jenkins01.arangodb.biz/job/arangodb-3.9-coverage/140/
https://jenkins01.arangodb.biz/job/arangodb-3.8-coverage/190/

**Review by KVS85 — APPROVED** [review]

**Pull request merged**